src/main/java/ch/uzh/ifi/hase/soprafs23/controller/GameController.java

```diff
@@ -0,0 +1,56 @@
+package ch.uzh.ifi.hase.soprafs23.controller;
+
+import ch.uzh.ifi.hase.soprafs23.entity.Game;
+import ch.uzh.ifi.hase.soprafs23.entity.Lobby;
+import ch.uzh.ifi.hase.soprafs23.rest.dto.GameGetDTO;
+import ch.uzh.ifi.hase.soprafs23.rest.dto.LobbyGetDTO;
+import ch.uzh.ifi.hase.soprafs23.rest.dto.TeamGetDTO;
+import ch.uzh.ifi.hase.soprafs23.rest.mapper.DTOMapper;
+import ch.uzh.ifi.hase.soprafs23.service.GameService;
+import ch.uzh.ifi.hase.soprafs23.service.LobbyService;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+
+/**
+ * User Controller
+ * This class is responsible for handling all REST request that are related to
+ * the user.
+ * The controller will receive the request and delegate the execution to the
+ * UserService and finally return the result.
+ */
+@RestController
+public class GameController {
+
+  private final GameService gameService;
+
+  GameController(GameService gameService) {
+    this.gameService = gameService;
+  }
+
+
+    @PostMapping("/games/{accessCode}")
+    @ResponseStatus(HttpStatus.CREATED)
+    @ResponseBody
+    public GameGetDTO createGame(@PathVariable int accessCode) {
```

Comment:
We changed the REST specification for the POST request of the game after M2. As far as I remember we decided to pass a Lobby object (containing access code, two teams of Users, ...) to the POST request.